docs/submission/submitting-molecular-data.md

```diff
@@ -121,93 +225,25 @@ The ARGO Data Platform accepts sequencing data submission for both `BAM` and `FA
 
 For both FASTQ and BAM submission, all files in the `files` section must be unique.
 
-### 1. Prepare Song sequencing_experiment metadata
-
-This is an example of a correctly formatted `sequencing_experiment` metadata submission in JSON format, according to the rules presented above:
+Example of files groups portion of payload:
 
 ```json
-{
-  "analysisType": {
-    "name": "sequencing_experiment"
-  },
-  "studyId": "TEST-CA",
-  "experiment": {
-    "submitter_sequencing_experiment_id": "EXP12345",
-    "sequencing_center": "OICR",
-    "platform": "ILLUMINA",
-    "platform_model": "HiSeq 2000",
-    "experimental_strategy": "WGS",
-    "sequencing_date": "2014-12-12"
-  },
-  "read_group_count": 3,
-  "read_groups": [
-    {
-      "submitter_read_group_id": "C0HVY.2",
-      "platform_unit": "74_8a",
-      "is_paired_end": true,
-      "file_r1": "test_rg3.bam",
-      "file_r2": "test_rg3.bam",
-      "read_length_r1": 150,
-      "read_length_r2": 150,
-      "insert_size": 232,
-      "sample_barcode": null,
-      "library_name": "Pond-147579"
-    },
-    {
-      "submitter_read_group_id": "D0RE2.1",
-      "platform_unit": "74_8b",
-      "is_paired_end": true,
-      "file_r1": "test_rg3.bam",
-      "file_r2": "test_rg3.bam",
-      "read_length_r1": 150,
-      "read_length_r2": 150,
-      "insert_size": 298,
-      "sample_barcode": null,
-      "library_name": "Pond-147580"
-    },
-    {
-      "submitter_read_group_id": "D0RH0.2",
-      "platform_unit": "74_8c",
-      "is_paired_end": true,
-      "file_r1": "test_rg3.bam",
-      "file_r2": "test_rg3.bam",
-      "read_length_r1": 150,
-      "read_length_r2": 150,
-      "insert_size": 298,
-      "sample_barcode": null,
-      "library_name": "Pond-147580"
-    }
-  ],
-  "samples": [
-    {
-      "submitterSampleId": "HCC1143_SMP1",
-      "matchedNormalSubmitterSampleId": "HCC1143_SMP2",
-      "sampleType": "Total DNA",
-      "specimen": {
-        "submitterSpecimenId": "HCC1143_SPC1",
-        "tumourNormalDesignation": "Tumour",
-        "specimenTissueSource": "Solid tissue",
-        "specimenType": "Primary tumour"
-      },
-      "donor": {
-        "submitterDonorId": "HCC1143",
-        "gender": "Female"
-      }
-    }
-  ],
-  "files": [
-    {
-      "fileName": "test_rg3.bam",
-      "fileSize": 14911,
-      "fileMd5sum": "178f97f7b1ca8bfc28fd5586bdd56799",
-      "fileType": "BAM",
-      "fileAccess": "controlled",
-      "dataType": "Submitted Reads"
-    }
-  ]
-}
+"files": [
+  {
+    "fileName": "test_rg3.bam",
+    "fileSize": 14911,
+    "fileMd5sum": "178f97f7b1ca8bfc28fd5586bdd56799",
+    "fileType": "BAM",
+    "fileAccess": "controlled",
+    "dataType": "Submitted Reads"
+  }
+]
 ```
 
+### 1. Prepare Song sequencing_experiment metadata
+
+This is an example of a correctly formatted `sequencing_experiment` metadata submission in JSON format, according to the rules presented above:

```

Comment:
Do we plan to add a complete example JSON payload here? Or we can add a link (for example payload) here to save some space.

Reply:
@lindaxiang i wondered the same thing.... maybe we could include a link later?

Reply:
@rosibaj , yes. I think a link to an example make it look nicer and neater.